server/events/project_finder_test.go

```diff
@@ -296,7 +296,7 @@ func TestDetermineProjects(t *testing.T) {
 	}
 	for _, c := range cases {
 		t.Run(c.description, func(t *testing.T) {
-			projects := m.DetermineProjects(noopLogger, c.files, modifiedRepo, c.repoDir, c.autoplanFileList)
+			projects := m.DetermineProjects(noopLogger, c.files, modifiedRepo, c.repoDir, c.autoplanFileList, nil)
```

Comment:
Do we need another test here?

Reply:
I don't think so? 

Relevant DetermineProjects code:

```go
		if moduleInfo != nil {
			downstreamProjects := moduleInfo.DependentProjects(path.Dir(modifiedFile))
			log.Debug("found downstream projects for %q: %v", modifiedFile, downstreamProjects)
			dirs = append(dirs, downstreamProjects...)
		}
```

and moduleInfo.DependentProjects is tested.